src/js/api/api-mutators.js

```diff
@@ -70,6 +70,7 @@ define([
             'setFullscreen',
             'setVolume',
             'setMute',
+            'setPlaylist',
```

Comment:
You're right, good call. 

I don't particularly like the alternative of passing in references to the controller into setup, but it's better than adding to the public API.